### DISCUSSION TITLE
Option in purpur.yml to grant ops all permissions

### DISCUSSION THREAD

**Discussion by 2Onkel3Fritz4** [PurpurMC/Purpur#1119] [__Dismissed__] [answered]
Because adding a new player to a Luckperms group with * permission is more complicated than using /op.
default it to false to avoid issues on servers already using permission plugins

**Comment by BillyGalbreath** [comment, marked as answer]
No.

> Luckperms group with * permission

That's horrible server administration. Please don't.
